crates/dprint/src/arg_parser.rs

```diff
@@ -195,6 +197,19 @@ fn inner_parse_args<TStdInReader: StdInReader>(args: Vec<String>, std_in_reader:
     ("editor-service", matches) => SubCommand::EditorService(EditorServiceSubCommand {
       parent_pid: matches.get_one::<String>("parent-pid").and_then(|v| v.parse::<u32>().ok()).unwrap(),
     }),
+    #[cfg(feature = "completions_generator")]
+    ("generate-completions", matches) => {
```

Comment:
Let's just call this "completions" to align with what `deno` does (`deno completions`)